package.json

```diff
@@ -15,11 +15,11 @@
   ],
   "author": "brian@hovercraftstudios.com",
   "license": "MIT",
-  "dependencies": {
-    "most": "^0.13.0"
-  },
   "devDependencies": {
-    "jshint": "~2.5",
-    "buster": "~0.7"
+    "jshint": "~2.9",
+    "buster": "~0.8"
+  },
+  "peerDependencies": {
+    "most": "^0.19.7"
```

Comment:
Hmmm, this is a tricky one.  Since `most-w3msg` depends on `most` *directly*, it's not clear what's right.  With npm 3, it's less of a problem, since it flattens the dep tree.  I guess switching to peerDeps ensures that folks don't end up with multiple copies of `most`.